app/build.gradle

```diff
@@ -60,5 +57,10 @@ dependencies {
     compile 'de.hdodenhof:circleimageview:2.1.0'
     compile 'com.jakewharton.timber:timber:4.5.1'
     compile 'com.jakewharton:butterknife:8.5.1'
+    compile 'com.android.support:support-v4:25.3.1'
```

Comment:
Please move the android support libraries to top and in 1 place. Similarly, test dependencies should be at top too.

Also, support-v4 is not needed as extra dependency. It's already included